source/computer/environment-variable.rst

```diff
@@ -0,0 +1,56 @@
+环境变量
+========
+
+当用户登录 Linux 系统后，Shell 程序启动并读取一系列配置文件，这些文件设置了可供所有用户共享的 Shell 环境。
+然后，Shell 会读取一些位于家目录下的配置文件，这些配置文件定义了用户个人的 Shell 环境。
+这些配置文件的名字和读取顺序在不同的 Shell 中或不同情况的 Shell 会话下，有所区别。
+Shell 环境由环境变量、Shell 变量、Shell 函数和别名等组成，这一节我们主要介绍环境变量。
+
+.. table:: Shell 的常见环境变量
+   :align: center
+
+   ================== =======================================
+   变量                内容    
+   ================== =======================================
+   SHELL              Shell 程序的名字
+   HOME               当前用户的家目录
+   LD_LIBRARY_PATH    程序加载运行期间，动态链接库的查找路径
+   PATH               由冒号分开的用于搜索可执行程序名的目录列表
+   PS1                Shell 提示符
+   PWD                当前工作目录
+   USER               当前用户的用户名
+   ================== =======================================
+
+使用  echo 命令查看环境变量::
+
+    # 查看 SHELL 变量，即 Shell 程序的名字
+    $ echo ${SHELL}
+
+使用 ``which`` 或 ``whereis`` 命令可查询命令所在目录::
+
+    $ which cat
+    /usr/bin/cat
+
+    $ whereis cat
+    cat: /usr/bin/cat /usr/share/man/man1/cat.1.gz /usr/share/man/man1p/cat.1p.gz
+
```

Comment:
It's unclear to me why we want to mention `which` and `whereis` here.

Reply:
I'd like to introduce those two commands in Linux101. Now I hid them in 49c6363.